modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/info/item/provider/ObjectEntryFolderInfoItemFields.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.object.web.internal.info.item.provider;
+
+import com.liferay.info.field.InfoField;
+import com.liferay.info.field.type.DateInfoFieldType;
+import com.liferay.info.field.type.TextInfoFieldType;
+import com.liferay.info.localized.InfoLocalizedValue;
+import com.liferay.object.model.ObjectEntryFolder;
+
+/**
+ * @author Marco Leo
+ */
+public class ObjectEntryFolderInfoItemFields {
+
+	public static final InfoField<DateInfoFieldType> createDateInfoField =
+		BuilderHolder._builder.infoFieldType(
+			DateInfoFieldType.INSTANCE
+		).name(
+			"createDate"
+		).labelInfoLocalizedValue(
+			InfoLocalizedValue.localize(
+				ObjectEntryFolderInfoItemFields.class, "create-date")
+		).build();
+	public static final InfoField<DateInfoFieldType> modifiedDateInfoField =
+		BuilderHolder._builder.infoFieldType(
+			DateInfoFieldType.INSTANCE
+		).name(
```

Comment:
Shouldn't this require alphabetical order? (labelInfoLocalizedValue-> name)

I've seen that in some places it is not ordered at all, so I'm not sure if this is necessary